docs/sphinx/using/examples/hardware_providers.rst

```diff
@@ -1,7 +1,7 @@
 Using Quantum Hardware Providers
 -----------------------------------
 
-CUDA-Q contains support for using a set of hardware providers (IonQ, IQM, OQC, and Quantinuum). 
+CUDA-Q contains support for using a set of hardware providers (IonQ, IQM, OQC, Quantinuum and ORCA Computing). 
```

Comment:
[consistency / nit]: These are in alphabetical order everywhere else..

Reply:
Sorted hardware_providers.rst and hardware.rst